src/main/java/com/github/tumbl3w33d/OAuth2ProxyHeaderAuthTokenFactory.java

```diff
@@ -47,7 +47,7 @@ public AuthenticationToken createToken(ServletRequest request, ServletResponse r
         } else {
             if (xForwardedUserHeader == null || xForwardedEmailHeader == null
                     || xForwardedPrefUsernameHeader == null) {
-                logger.warn("required OAuth2 proxy headers incomplete - {}: {} - {}: {} - {}: {}",
+                    logger.debug("required OAuth2 proxy headers incomplete - {}: {} - {}: {} - {}: {}",
```

Comment:
Wrong indent?

Reply:
Save action didn't seem to care, manually running the formatter does. 🤷🏻‍♂️